Source/SuperLinq/Move.cs

```diff
@@ -32,21 +32,30 @@ public static partial class SuperEnumerable
 	/// <remarks>
 	///	    This operator uses deferred execution and streams its results.
 	/// </remarks>
-	public static IEnumerable<T> Move<T>(this IEnumerable<T> source, int fromIndex, int count, int toIndex)
+	public static IEnumerable<T> Move<T>(
+		this IEnumerable<T> source,
+		int fromIndex,
+		int count,
+		int toIndex
+	)
 	{
 		ArgumentNullException.ThrowIfNull(source);
 		ArgumentOutOfRangeException.ThrowIfNegative(fromIndex);
 		ArgumentOutOfRangeException.ThrowIfNegative(count);
 		ArgumentOutOfRangeException.ThrowIfNegative(toIndex);
 
-		return
-			toIndex == fromIndex || count == 0
-				? source :
-			toIndex < fromIndex
-				 ? Core(source, toIndex, fromIndex - toIndex, count)
-				 : Core(source, fromIndex, count, toIndex - fromIndex);
+		return toIndex == fromIndex || count == 0
```

Comment:
The desired format for nested ternaries in this repository is to have each condition and its outcome on its own line. For your use case, this would look like:

```cs
return conditionOne ? outcomeOne : 
    conditionTwo ? outcomeTwo : 
    outcomeThree;
```